### ISSUE TITLE
Fix scanner eating SECTIONs

### ISSUE THREAD

**Issue by GitMensch** [mridoni/gixsql#36]
It _is_ the scanner - https://github.com/mridoni/gix/blob/61349a5b0e6d349518cdbaa16f03b53d3cc8b55f/libgixpp/gix_esql_scanner.ll#L145

You should be able to see it when applying the following diff to your sample:

```diff
        PROCEDURE DIVISION. 
-       
+       DO-FILE SECTION.
        000-CONNECT.
```

Similar things would then happen with `HANDLE-LINKAGE SECTION.`

_Originally posted by @GitMensch in https://github.com/mridoni/gixsql/issues/3#issuecomment-1099959128_

**Comment by GitMensch** [comment]
Also highlighting: this is a really nasty issue ;-)

**Comment by mridoni** [comment]
Just pushed a commit that, at least in my test case, solves this. The test case is `TSQL007A.cbl`

**Comment by GitMensch** [comment]
Thank you. Will try to check as soon as possible - which likely still means "with the next release".

In my point of view the -ARR/-LEN suffix and #8 and, if reproducible #1 are the only _necessary_ for it, do you plan other things? 

More is nice of course, but "time" is often an issue, as I know...

BTW: Please dare to "fix #NN" in your commits ;-)

**Comment by GitMensch** [comment]
That one is definitely solved for 1.15!
